packages/server/graphql/mutations/endRetrospective.ts

```diff
@@ -120,10 +120,10 @@ export default {
       dataLoader.get('teams').load(teamId),
       dataLoader.get('teamMembersByTeamId').load(teamId),
       removeEmptyTasks(meetingId),
-      dataLoader.get('meetingTemplates').load(templateId)
+      dataLoader.get('meetingTemplates').load(templateId),
+      finishRetroMeeting(completedRetrospective, dataLoader)
```

Comment:
this isn't safe because we need to wait for `removeEmptyTasks` probably worth reverting the change & adding a comment as to why it's done in serial

Reply:
Ahh yes! I guess this is the same for [check-ins too](https://github.com/ParabolInc/parabol/blob/8d3826bfaedeb8d17b08afbd9bda56866eace4f6/packages/server/graphql/mutations/endCheckIn.ts#L231) where we're removing empty tasks and finishing the meeting in the same `Promise.all`. I've moved `finishCheckInMeeting` too and added comments in both files 👍